### PR TITLE
autoscroll to job running

### LINKED ISSUE #709: Job Progress - auto scroll
- [ ] JobModal should vertically center the active task

### PR DESCRIPTION
**Pull request by ramirobg94**
resolves #709

### CODE REVIEW

File: webapp/views/App/JobMonitor/InnerJobs/InnerJob/InnerJob.js [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+
+import JobProgress from '../../JobProgress'
+import JobErrors from '../../JobErrors'
+
+const InnerJob = ({ innerJob, i }) => {
+  const i18n = useI18n()
+  const elementRef = useRef(null)
+
+  useEffect(() => {
+    if (JobSerialized.getStatus(innerJob) === 'running' && elementRef.current) {
```
Comment (SteRiccio) [inline webapp/views/App/JobMonitor/InnerJobs/InnerJob/InnerJob.js:16]: `JobSerialized.isRunning(innerJob)` can be used instead

File: webapp/views/App/JobMonitor/InnerJobs/InnerJob/InnerJob.js [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+        <JobProgress job={innerJob} />
+      </div>
+      <JobErrors job={innerJob} />
+    </>
+  )
+}
+
+InnerJob.propTypes = {
+  innerJob: PropTypes.object.isRequired,
+  i: PropTypes.number.isRequired,
```
Comment (SteRiccio) [inline webapp/views/App/JobMonitor/InnerJobs/InnerJob/InnerJob.js:36]: can it be renamed into `index` to make it more explicit?

File: webapp/views/App/JobMonitor/InnerJobs/InnerJob/InnerJob.js (hunk 1 above)
Comment (ramirobg94) [reply webapp/views/App/JobMonitor/InnerJobs/InnerJob/InnerJob.js:16]: ouuu i will change!

File: webapp/views/App/JobMonitor/InnerJobs/InnerJob/InnerJob.js (hunk 2 above)
Comment (ramirobg94) [reply webapp/views/App/JobMonitor/InnerJobs/InnerJob/InnerJob.js:36]: sure!

**Pull request merged**